<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtWidgetsMessageBox2</class>
 <widget class="QWidget" name="QtWidgetsMessageBox2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>提示</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="pushButton_OK">
   <property name="geometry">
    <rect>
     <x>236</x>
     <y>205</y>
     <width>190</width>
     <height>48</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color:rgb(57, 59, 64);
color:rgb(244, 216, 168);
border:none;
border-radius:4px;
}
QPushButton:hover {
background-color:rgb(77, 79, 83);
}
QPushButton:pressed {
background-color:rgb(57, 59, 64);
}</string>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Message">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>115</y>
     <width>400</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>-1</pointsize>
     <stylestrategy>PreferDefault</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color:rgb(57, 59, 64);
font-size:18px;
}</string>
   </property>
   <property name="text">
    <string>更新未完成，是否继续退出？</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="BackgrouundImageRect">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>460</width>
     <height>280</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
background-image:url(:/BackgroundImage/resource/QMessageBox/bg.png);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Cancel">
   <property name="geometry">
    <rect>
     <x>34</x>
     <y>205</y>
     <width>190</width>
     <height>48</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color:rgb(255, 255, 255);
color:rgb(220, 188, 96);
border:1px solid rgb(204, 204, 204);
border-radius:4px;
}
QPushButton:hover {
background-color:rgb(251, 248, 239);
}
QPushButton:pressed {
background-color:rgb(236, 233, 224);
}</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="TitleLabelRect">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>460</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>BackgrouundImageRect</zorder>
  <zorder>pushButton_OK</zorder>
  <zorder>label_Message</zorder>
  <zorder>pushButton_Cancel</zorder>
  <zorder>TitleLabelRect</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtWidgetsMessageBox.qrc"/>
  <include location="TianLiInstallationPackage.qrc"/>
 </resources>
 <connections/>
</ui>
